portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,48 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List; // import just the List interface
+import java.util.ArrayList; // import just the ArrayList class
```

Comment:
Nit: We can remove the import comments here. The import statements themselves are self-documenting.

Reply:
removed